roles/test_sensubility/tasks/test_health_status.yml

```diff
@@ -71,9 +66,8 @@
     cmd: >-
       /usr/bin/curl -k {{ prom_auth_string }} \
         -g https://{{ prom_url }}/api/v1/query? \
-        --data-urlencode 'query=last_over_time(sensubility_container_health_status{process="logrotate_crond",host="controller-0.redhat.local"}[10m])' \
-        | grep -oP '(?<="value":).*' | awk -F, '{ print $2 }' |  grep -o '[0-9]\+' | grep 1 | wc -l
+        --data-urlencode 'query=last_over_time(sensubility_container_health_status{process="logrotate_crond",host="ceph-0.redhat.local"}[10m])' \
```

Comment:
This change seems unrelated

Reply:
This comes from here: https://github.com/infrawatch/feature-verification-tests/pull/180

I believe this shows up because of the order in which these changes were proposed.

Reply:
I've targetted this change against PR#180, instead of master. I think that should reflect the correct changes for this PR, as well as represent the dependency